src/test/resources/edu/hm/hafner/analysis/parser/CodeChecker_with_linux_paths.txt

```diff
@@ -24,6 +24,16 @@ Found 1 defect(s) in workbook.cpp
 
 Found 1 defect(s) in HPSF.cpp
 
+[UNSPECIFIED] /path/to/projrct/extern/lib/dem.c:1778:7: misra violation (use --rule-texts=<file> to get proper output) [cppcheck-misra-c2012-15.5]
+      return E_NOT_OK;
+      ^
```

Comment:
Note: lines 28-29 are currently ignored. If this information is useful it might make sense to improve the parser in an additional PR.

Reply:
I initially thought i'd like CodeChecker, but.. no so sure anymore. So might get thrown out in this project. Then i won't spend any more time on its support :). These are still valid changes thou.